poetry.lock

```diff
@@ -705,7 +705,7 @@ testing = ["pytest", "pytest-benchmark"]
 
 [[package]]
 name = "pminit"
-version = "0.3.1.dev37+f1e980c"
+version = "0.5.1.dev152+7b661a4"
 description = "Post-install hook for PythonMonkey"
 optional = false
 python-versions = "^3.8"
```

Comment:
I think we'd better remove the pminit entries from poetry.lock entirely. If the current version is higher than this, `poetry install` give a very strange "upgrade to a lower version" each time it's run. 
![](https://github.com/Distributive-Network/PythonMonkey/assets/16057139/2e6fb6d2-d3eb-4218-98ba-4e4bad1bc716)

Upgrading pminit from `0.5.1.dev72+6296419` to `0.3.1.dev37+f1e980c`. What?

Reply:
Huh, odd. Will do!

Reply:
Ah, but that then leads to: 
```
(pythonmonkey-rf3EeaoG-py3.11) caleb@bergelmir:~/git/PythonMonkey$ poetry install
Installing dependencies from lock file
Warning: poetry.lock is not consistent with pyproject.toml. You may be getting improper dependencies. Run `poetry lock [--no-update]` to fix it.
```
which is preferable?

Reply:
What changes did you make?

Reply:
![image](https://github.com/Distributive-Network/PythonMonkey/assets/82386504/6ac24131-b1e5-4196-99ed-b16f4f398619)

Reply:
When removing the thing entirely, it doesn't give me the warning.
![image](https://github.com/Distributive-Network/PythonMonkey/assets/16057139/5fe98ca4-9878-46f6-abd5-9713b78a0fbf)

Reply:
pushed recommended changes, I think your way is still preferable

Reply:
> Ah, but that then leads to:
> 
> ```
> (pythonmonkey-rf3EeaoG-py3.11) caleb@bergelmir:~/git/PythonMonkey$ poetry install
> Installing dependencies from lock file
> Warning: poetry.lock is not consistent with pyproject.toml. You may be getting improper dependencies. Run `poetry lock [--no-update]` to fix it.
> ```
> 
> which is preferable?

@caleb-distributive Now I can recall the reason why I did put a `*` instead of a specific version in the first place when I worked on this.